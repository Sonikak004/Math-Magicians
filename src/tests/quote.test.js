// QuoteComponent.test.js
import React from 'react';
import { render, screen } from '@testing-library/react';
import QuoteComponent from '../components/quote';
import '@testing-library/jest-dom/extend-expect';

describe('QuoteComponent', () => {
  test('renders correctly', () => {
    render(<QuoteComponent />);
    const quoteElement = screen.getByTestId('quote');
    expect(quoteElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const { asFragment } = render(<QuoteComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
