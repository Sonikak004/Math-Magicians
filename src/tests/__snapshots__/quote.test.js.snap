// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteComponent matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="quote font"
    data-testid="quote"
  >
    Loading...
  </div>
</DocumentFragment>
`;
